### DISCUSSION TITLE
PnP - Search Box is not working wirh PnP - Search Result

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2801] [Q&A]
I have added a PnP - Seach Box web part, with the following settings:-

![image](https://user-images.githubusercontent.com/79726511/220995352-028e0c40-0a03-4e49-986f-52ae73b13a95.png)

![image](https://user-images.githubusercontent.com/79726511/220995394-3ae75112-b071-47ea-b293-7e71360a8879.png)

![image](https://user-images.githubusercontent.com/79726511/220995432-269060e0-de70-4c5d-9a9d-58c08e21fc44.png)

and i linked it with the PnP - Search Result:-

![image](https://user-images.githubusercontent.com/79726511/220995600-5b381ccb-88d4-4335-a5b5-4dd624fc72a9.png)

and inside the Query template  i defined to pass the SearchTerm as follow:-

![image](https://user-images.githubusercontent.com/79726511/220995723-dd21ad3c-0bac-4016-8004-4e692a617865.png)

but currently when i type some text inside the search box and i click on search nothing will get reflected inside the PnP - Search Result web part... any advice?

**Comment by wobba** [comment]
I suggest you go thru scenario tutorials at https://microsoft-search.github.io/pnp-modern-search/scenarios/
